liwords-ui/src/gameroom/table.tsx

```diff
@@ -79,8 +75,19 @@ export const Table = (props: Props) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
-  const player1 = fullPlayerInfo(0, gameState);
-  const player2 = fullPlayerInfo(1, gameState);
+  // Figure out what rack we should display.
+  // If we are one of the players, display our rack.
+  // If we are NOT one of the players (so an observer), display the rack of
+  // the player on turn.
+  let rack;
+  const us = gameContext.players.find((p) => p.nickname === props.username);
+  if (us) {
+    rack = us.currentRack;
+  } else {
+    rack = gameContext.players.find((p) => p.onturn)?.currentRack || '';
+  }
+
+  console.log('in table', gameContext.players);
```

Comment:
Why are we only displaying one of the racks? Why not both? This is one of the things that annoys me about ISC.

Reply:
I agree, I also want to display both racks, but Conrad wetoed it as he was concerned it made it too easy to cheat by having a watcher on the table. We could discuss, I think this would be better as well.